Source/CombatExtended/CombatExtended/Projectiles/ProjectilePropertiesCE.cs

```diff
@@ -30,6 +30,11 @@ public class ProjectilePropertiesCE : ProjectileProperties
     public float armorPenetrationBlunt;
     public bool castShadow = true;
 
+    public float recoilMultiplier = 1;
+    public float warmupMultiplier = 1;
+    public float effectiveRangeMultiplier = 1;
+    public float muzzleFlashModifier = 0f;
```

Comment:
If this is additive, I'd probably call it `muzzleFlashOffset`, to keep with typical RW stat naming conventions and make the functionality clearer.

Reply:
Makes sense, renamed